src/app/components/Button/Button.module.css

```diff
@@ -4,19 +4,22 @@
   color: var(--button-text-color);
   font-family: var(--button-font);
   font-size: 1.25em;
+  width: 100%;
 }
 
 .button--primary {
   background-color: var(--secondary-color);
   padding: 1em 4em;
   border-radius: 3em;
+  width: 100%;
```

Comment:
If `button--primary` is a modifier of `button`, you don't need to add width here (and in `button--secondary`)

Reply:
You're right! However, I deleted it from the button component, because only the button-primary and -secondary need a width of 100%.